README.md

```diff
@@ -8,305 +8,409 @@
 
 <!-- END SHOW operator hub -->
+[Back to top](#ibm-cloud-operator)
+
+## Using separate IBM Cloud accounts
+
+You can provision IBM Cloud services in separate IBM Cloud accounts from the same cluster. To use separate accounts, update the secrets and configmap in the Kubernetes namespace where you want to create services and bindings. 
+
+**Tip**: Just want to use a different account one time and don't want to manage a bunch of namespaces? You can also specify a different account in the individual [service configuration](#service-properties), by overriding the default [account context](#account-context-in-operator-secret-and-configmap).
+
+1.  Get the IBM Cloud account details, including account ID, Cloud Foundry org and space, resource group, region, and API key credentials.
+2.  Edit or replace the `ibmcloud-operator-secret` secret in the Kubernetes namespace that you want to use to create services in the account.
+3.  Edit or replace the `ibmcloud-operator-defaults` configmap in the Kubernetes namespace that you want to use to create services in the account.
+4.  Optional: [Set up a management namespace](#setting-up-a-management-namespace) so that cluster users with access across namespaces cannot see the API keys for the different IBM Cloud accounts. 
+
+[Back to top](#ibm-cloud-operator)
+
+## Setting up a management namespace
+
+By default, the API key credentials and other IBM Cloud account information are stored in a secret and a configmap within each namespace where you create IBM Cloud Operator service and binding custom resources. However, you might want to hide access to this information from cluster users in the namespace. For example, you might have multiple IBM Cloud accounts that you do not want cluster users in different namespaces to know about.
+
+1.  Create a management namespace that is named `safe`.
+    
+    ```bash
+    kubectl create namespace safe
+    ```
+
+2.  In the namespace where the IBM Cloud Operator runs, create an `ibmcloud-operator-config` configmap that points to the `safe` namespace.
+    
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: ibmcloud-operator-config
+      namespace: ibmcloud-operators # Or update to the namespace where IBM Cloud Operator is running
+      labels:
+        app.kubernetes.io/name: ibmcloud-operator
+    data:
+      namespace: safe
+    EOF
+    ```
+
+3.  Copy the existing or create your own `ibmcloud-operator-secret` secrets and `ibmcloud-operator-defaults` configmaps from the other Kubernetes namespace into the `safe` namespace. **Important**: You must rename the secrets and configmaps with the following naming convention. **Tip**: You can create these new secrets and configmaps similarly to `make install`, if you are familiar with that process.
```

Comment:
Maybe we should remove the `make install` tip? I think that may confuse users, since it's more of a development thing.